tests/js/spec/views/organizationIntegrations/integrationRepos.spec.jsx

```diff
@@ -15,6 +15,92 @@ describe('IntegrationRepos', function() {
     const integration = TestStubs.GitHubIntegration();
     const routerContext = TestStubs.routerContext();
 
+    describe('successful save', async function() {
+      let addRepo = Client.addMockResponse({
+        url: `/organizations/${org.slug}/repos/`,
+        method: 'POST',
+        body: {
```

Comment:
Can we use anything in `tests/js/fixtures`?

Reply:
The fixture in `tests/js/fixtures/repository.js` gets close. I'll use that.